src/add&delete.js

```diff
@@ -0,0 +1,77 @@
+import isChecked from './updateStatus';
+import editTask from './editTask';
+import clearTasks from './clearTasks'; // eslint-disable-line import/no-cycle
+
+const submitForm = document.querySelector('form');
+const taskInput = document.querySelector('#task-input');
+const todoList = document.querySelector('.list-group');
+
+const printTask = (task) => {
```

Comment:
I think it is better if you use classes over ES5 functions, since classes are the new syntactic sugar in JavaScript. What do you think?